src/components/progress-tracker/progress-tracker.style.ts

```diff
@@ -79,16 +85,22 @@ const StyledDescription = styled.span`
 `;
 
 const StyledValuesLabel = styled.span<
-  Pick<ProgressTrackerProps, "size" | "labelsPosition">
+  Pick<ProgressTrackerProps, "size" | "labelsPosition" | "labelWidth">
 >`
   text-align: start;
   display: flex;
   justify-content: flex-start;
   gap: 4px;
   font-size: ${({ size }) => size && fontSizes[size]};
   ${({ labelsPosition }) => labelsPosition === "bottom" && "margin-top: 8px"};
```

Comment:
suggestion: you could refactor this to destructure it once rather than three times if you want (non-blocking). We should use the token value though.

```
  const labelsPositionMargin = { top: "bottom", bottom: "top", left: "right" };

 ...
 const StyledValuesLabel = styled.span<
  Pick<ProgressTrackerProps, "size" | "labelsPosition" | "labelWidth">
>`
...
  ${({ labelsPosition }) => `margin-${labelsPositionMargin[labelsPosition]}: var(--spacing100)`};
```